packages/connect-explorer/src/actions/trezorConnectActions.ts

```diff
@@ -160,20 +163,39 @@ export const init =
         };
 
         try {
-            await TrezorConnect.init(connectOptions);
+            if (deeplink) {
+                await TrezorConnectDeeplink.init({
+                    ...connectOptions,
+                    deeplinkOpen(url) {
+                        window.open(url, '_blank');
+                    },
+                    deeplinkCallbackUrl:
+                        (process.env.CONNECT_EXPLORER_FULL_URL || window.location.origin) +
+                        '/callback',
+                });
+                const bc = new BroadcastChannel('trezor_connect_callback');
```

Comment:
what is this good for? you are not passing `bc` anywhere so I guess no one is subscribed to that messages anyway

Reply:
The callback page opens a BroadcastChannel with the same ID and through this it's able to send the response back to the main tab.

Reply:
Thanks for explanation, I was not familiar with that API